### DISCUSSION TITLE
I am unable to add DxCore

### DISCUSSION THREAD

**Discussion by Clement1180** [SpenceKonde/DxCore#533] [Q&A]
Hello,
I tried several versions of the Arduino IDE (2.3.2 & 1.8.19) and I still have the same problem when I try to download the package with this link: http://drazzy.com/package_drazzy.com_index.json.

 I have the following error:

 Some indexes could not be updated. Get "https://drazzy.com/package_drazzy.com_index.json": tls: failed to verify certificate: x509: certificate has expired or is not yet valid: : http://drazzy.com/package_drazzy.com_index.json. 

How can I solve this problem?

**Comment by mechatroniks-git** [comment]
See this issue for possible workaround.

https://github.com/SpenceKonde/megaTinyCore/issues/1106
